tests/test_routing_lievre_tutorial.py

```diff
@@ -234,3 +236,235 @@ def test_lievre_tutorial(self):
             (4000, 129.98874279175033),
         ]:
             assert model.hydrograph.q_sim[d].item() == pytest.approx(q_sim)
+
+    def test_lievre_tutorial_v21(self):
+        """
+        This test reproduces the Lievre tutorial setup (with the Routing Product V2.1):
+
+        http://raven.uwaterloo.ca/files/RavenTutorial6.zip
+
+        """
+
+        ###############
+        # Input files #
+        ###############
+
+        routing_product_shp_path = get_local_testdata(
+            "raven-routing-sample/lievre_hrus_v21.zip"
+        )
+
+        vic_streaminputs_nc_path = get_local_testdata(
+            "raven-routing-sample/VIC_streaminputs.nc"
+        )
+        vic_temperatures_nc_path = get_local_testdata(
+            "raven-routing-sample/VIC_temperatures.nc"
+        )
+
+        observation_data_nc_path = get_local_testdata(
+            "raven-routing-sample/WSC02LE024.nc"
+        )
+
+        #########
+        # Model #
+        #########
+
+        model = Raven()
+
+        model.config.identifier = "raven-lievre-routing-v21"
+
+        #######
+        # RVI #
+        #######
+
+        model.config.rvi.tmpl = """
+        :CatchmentRoute        ROUTE_DUMP                    # Catchment routing method, used to convey water from the catchment tributaries and rivulets to the subbasin outlets. DEFAULT ROUTE_DUMP, which instantly ‘dumps’ all water in the subbasin stream reach.
+        :Routing               ROUTE_DIFFUSIVE_WAVE          # Channel routing method which is used to transport water from upstream to downstream within the main subbasin channels. DEFAULT ROUTE_DIFFUSIVE_WAVE, which analytically solves the diffusive wave equation along the reach using a constant reference celerity.
+        :PrecipIceptFract      PRECIP_ICEPT_NONE             # Estimation of the precipitation interception fraction. In this routing model, stream input(s) are "pretending" to be precipitation going into Raven, thus using DEFAULT PRECIP_ICEPT_NONE to indicate no interception processes are adopted.
+        :PotentialMeltMethod   POTMELT_NONE                  # Estimation of the potential snow melt. In this routing model, snow melt processes are not relevant, thus using DEFAULT POTMELT_NONE method.
+        :SoilModel             SOIL_ONE_LAYER                # In this routing model, use DEFAULT SOIL_ONE_LAYER to define single soil layer structure.
+
+        :HydrologicProcesses
+          :Precipitation     PRECIP_RAVEN             ATMOS_PRECIP     PONDED_WATER          # Moves stream input(s) from ATMOS_PRECIP to PONDED_WATER storage (waiting for runoff). Use DEFAULT PRECIP_RAVEN method.
+          :Flush             RAVEN_DEFAULT            PONDED_WATER     SURFACE_WATER         # Moves water from PONDED_WATER to SURFACE_WATER (routed to outlet). Use DEFAULT RAVEN_DEFAULT method.
+        :EndHydrologicProcesses
+        """
```

Comment:
Could this be replaced with the Routing emulator ?

Reply:
I'm not sure what you mean as I have removed the routing emu a while ago.. it wasn't really an emulator per se no?

Reply:
I hadn't realized this. You're right that it's a bit different to other emulators, but I thought it was nice to have it around.

Reply:
I did not understood in what sense it was actually an emulator.. it didn't have any particular parameter, etc. If you want it back I'm happy to discuss it though.

Reply:
I don't have strong feelings about it. It may have made the test code here a bit simpler, but I think it may be confusing to users if there's an emulator that is substantially different from others. Let's leave it at that for now.